backend/src/main/java/ddangkong/domain/balance/room/Room.java

```diff
@@ -3,27 +3,51 @@
 import ddangkong.exception.BadRequestException;
 import jakarta.persistence.Column;
 import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
+import lombok.AccessLevel;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 
 @Entity
 @Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
 public class Room {
 
     private static final int DEFAULT_TOTAL_ROUND = 5;
+    private static final int DEFAULT_TIME_LIMIT = 10000;
```

Comment:
```suggestion
    private static final int DEFAULT_TIME_LIMIT_MS = 10000;
```
사소한 의견) 시간 단위가 변수명에 나타나면 좋겠으나, 방 설정 정보를 받는 것은 추후 구현되기 때문에 곧 사라질 상수라는 생각이 드네요 편하게 채택해주세요!

Reply:
해당 값 사용과 연결되는 방 설정값 적용 API 또한 이번 스프린트3에서 제 담당입니다만, 방 생성 시에 기본값은 그대로 가져갈 것 같습니다!
때문에 이해하기 편하도록 프린 의견 바로 변경해두겠습니다:)